test/library/standard/Map/testToArray.chpl

```diff
@@ -1,6 +1,8 @@
 use Map;
 
-var m = new map(int, int);
+config param parSafe = false;
+
+var m = new map(int, int, parSafe);
 for i in -10..10 do
   m[i] = -i;
```

Comment:
Could you please change this line to `m.add(-i);`? This operation is no longer permitted when `parSafe=true`.